tailor-app/src/main/java/com/future/tailormade/tailor_app/core/model/ui/orderDetail/OrderDetailUiModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.future.tailormade.tailor_app.core.model.ui.orderDetail
+
+import com.future.tailormade.tailor_app.core.model.ui.order.OrderDesignUiModel
+
+data class OrderDetailUiModel(
+
+    var orderDate: String,
+
+    var design: OrderDesignUiModel,
+
+    var id: String,
+
+    var measurement: OrderDetailMeasurementUiModel,
+
+    var quantity: String,
+
+    var specialInstructions: String? = null,
+
+    var status: String,
+
+    var totalDiscount: String,
```

Comment:
total disocunt, totalPricem and paymentTotal should be in double

Reply:
This value has been mapped to String in UiModel to set the text directly kak
Any concern kak?

Reply:
Has been resolved